_applications/equoid.adoc

```diff
@@ -0,0 +1,223 @@
+    -p MANAGEMENT_USER=radlee \
+    -p MANAGEMENT_PASSWORD=radlee
+....
++
+
+. Launch Data Publisher
++
+We can next begin publishing our streaming data by starting an instance of the data publisher. `OP_MODE` described later should be set to the mode you wish. Also, `DATA_URL_PRIMARY` should be set to the location of your data set or data potential values file. For purposes of this example we'll use linear processing of a full dataset contained in the `equoid-data-publisher` github repo.  
++
+....
+oc new-app \
+    -l app=publisher \
+    -e OP_MODE=linear \
+    -e DATA_URL_PRIMARY=https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/StockCodesLinear.txt  \
+    --image-stream=`oc project -q`/fuse-java \
+    https://github.com/radanalyticsio/equoid-data-publisher
+....
+
+. Launch Data Handler
++
+We are now ready to start our data handler. Note that `INFINISPAN_HOST` and `INFINISPAN_PORT` need to be set to appropriate values, in the case of the template provided, these are `datagrid-hotrod` and `11222` respectively. In addition, `WINDOW_SECONDS` is the number of second for which you wish to track the top-k items, `SLIDE_SECONDS` represents the number of seconds to slide your data window each iteration, and `BATCH_SECONDS` is the size of samples you wish to acquire from the message stream at a time. Finally, `OP_MODE` needs to be set to the same value as set for the data publisher. In our example we will use linear corresponding with our previous setting for the data publisher, and 20 seconds for the window, slide, and batch sized. 
++
+....
+oc new-app --template=oshinko-scala-spark-build-dc \
+    -l app=handler \
+    -p SBT_ARGS=assembly \
+    -p APPLICATION_NAME=equoid-data-handler \
+    -p GIT_URI=https://github.com/radanalyticsio/equoid-data-handler \
+    -p APP_MAIN_CLASS=io.radanalytics.equoid.DataHandler \
+    -e INFINISPAN_HOST=datagrid-hotrod \
+    -e INFINISPAN_PORT=11222 \
+    -e WINDOW_SECONDS=20 \
+    -e SLIDE_SECONDS=20 \
+    -e BATCH_SECONDS=20 \
+    -e OP_MODE=linear \
+    -p SPARK_OPTIONS='--driver-java-options=-Dvertx.cacheDirBase=/tmp'
+....
+
+****
+The data publisher and data handler can take several minutes to build from source. If in doubt of the build progress, from the command line you can run `oc logs -f bc/equoid-data-publisher` and `oc logs -f bc/equoid-data-handler` or look in the two components respective sections of the OpenShift UI (e.g. browse to project overview, click equoid-data-publisher or handler application arrow to expand and display build status) 
+****
+
+. Launch Web UI and Set Up Keycloak
++
+Equoid's web UI is launched using a script developed by one of Equoid's contributors, Jiri Kremser. The following calls will utilize that script to set up the UI as well as a Keycloak instance for users and login. Finally, an edit role is given to the project's service account so additional data handler instances can be created/modified. 
```

Comment:
? for users and login --> for users to login?

Reply:
Might have been clearer if I put "for user accounts and login credentials" but I like "for users to login" better, it's simpler and gets the point across. Added to revision, thanks